src/sagemaker/model_monitor/model_monitoring.py

```diff
@@ -453,6 +453,10 @@ def update_monitoring_schedule(
         network_config_dict = None
         if self.network_config is not None:
             network_config_dict = self.network_config._to_request_dict()
+            if "EnableInterContainerTrafficEncryption" in network_config_dict:
+                raise ValueError(
+                    "EnableInterContainerTrafficEncryption is not supported in Model Monitor"
```

Comment:
let's make this more actionable, e.g. `"EnableInterContainerTrafficEncryption is not supported in Model Monitor. Please ensure that encrypt_inter_container_traffic=None when creating your NetworkConfig object."`

You could even add something like `"Current encrypt_inter_container_traffic value: {}".format(self.network_config.encrypt_inter_container_traffic)`

Reply:
Done